pulp_manifest/setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='pulp_manifest',
-    version='1.0.0',
```

Comment:
I'm not sure if there's a reason this one was versioned  differently, so I will defer judgement.

Reply:
It's only versioned differently b/c it was added very recently (after 2.15 went out, before 2.16) and should follow lock-stepping iirc.  The rpm's version will be the same alongside the rest of pulp-rpm

@werwty does that sound correct?

Reply:
@pcreech that sounds correct